Jenkinsfile

```diff
@@ -660,9 +660,15 @@ def withBeatsEnv(boolean archive, Closure body) {
   ]) {
     deleteDir()
     unstash 'source'
-    dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
-    // FIXME workaround untill we fix the packer cache
-    sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
+    if(os == 'linux'){
```

Comment:
We have an arm64 worker now that does not have Docker. So this test for "os == linux" probably isn't sufficient. Is there another way to implement this?

Reply:
yep, it is better to check if docker is installed instead of base it on the OS, when we remove the workaround we fix it too